dazel.py

```diff
@@ -188,15 +188,18 @@ def send_command(self, args):
             return os.WEXITSTATUS(rc)
 
     def _fix_win_symlink(self, docker_exec_command):
-        p = pathlib.Path(self.directory)
-        for path in list(p.glob('bazel-*')):
-            command = "%s realpath %s" % (
-                docker_exec_command, str(path.name))
-            output = self._run_command(command).strip()
-            drive = pathlib.PureWindowsPath(path.drive)
-            local_directory = drive.joinpath(pathlib.PurePosixPath(output))
-            path.unlink()
-            path.symlink_to(local_directory, target_is_directory=True)
+        try:
+            p = pathlib.Path(self.directory)
+            for path in list(p.glob('bazel-*')):
+                command = "%s realpath %s" % (
+                    docker_exec_command, str(path.name))
+                output = self._run_command(command).strip()
+                drive = pathlib.PureWindowsPath(path.drive)
+                local_directory = drive.joinpath(pathlib.PurePosixPath(output))
+                path.unlink()
+                path.symlink_to(local_directory, target_is_directory=True)
+        except:
```

Comment:
This exception is a bit too broad. What exception do you expect here? can you catch the specific thing?
Also - besides pass please also explain why it's ok to ignore the exception here.

Reply:
Yes, this was badly done. Fixed now.